numpy/add_newdocs.py

```diff
@@ -5446,7 +5446,9 @@ def luf(lamdaexpr, *args, **kwargs):
         Input arrays.
     out : ndarray or tuple of ndarray, optional
         Alternate array object(s) in which to put the result; if provided, it
-        must have a shape that the inputs broadcast to.
+        must have a shape that the inputs broadcast to. A tuple of arrays
+        (possible only as a keyword argument) must have length equal to the
+        number of outputs; unwanted outputs can be set to `None`.
```

Comment:
`None` doesn't mean "I don't want an output", just "you can allocate it yourself"